TestResultSummaryService/perf/DataManagerPerf.js

```diff
@@ -0,0 +1,93 @@
+const { TestResultsDB, OutputDB } = require( '../Database' );
+const ObjectID = require( 'mongodb' ).ObjectID;
+const Parsers = require( `../parsers/` );
+const DefaultParser = require( `../parsers/Default` );
+const { logger } = require( '../Utils' );
+const BenchmarkMath = require( './BenchmarkMathCalculation' );
+const math = require('mathjs');
+
+class DataManagerPerf {
+    async updateBuildWithAggResult ( task ) {
+        if ( task.type === "Perf") {
```

Comment:
Yeah this may not work in all cases. For perf, the root build would be the grandparent build and then we'll have 2 more levels: parent and child builds. As Lan mentioned, parent build might not be `Perf`.